lib/api/controllers/serverController.js

```diff
@@ -110,6 +112,28 @@ class ServerController extends NativeController {
     return config;
   }
 
+  /**
+   * Returns the Kuzzle capabilities
+   * @returns {Promise<Object>}
+   */
+  async capabilities () {
+    const config = JSON.parse(JSON.stringify(global.kuzzle.config));
+    const publicApi = await this.publicApi();
+    const services = [];
+
+    _.map(config.services, (val, key) => {
+      if (val.backend !== undefined) {
+        services.push({ [key]: val.backend });
+      }
+    });
+
+    return {
+      limits: config.limits,
+      routes: publicApi,
+      services: services,
```

Comment:
You should not expose the entire `services` configuration object, it contains sensitive values.

Here we want to know what kind of services Kuzzle is using: Elasticsearch or Postgres? Redis or RAM ?

Reply:
This is what I do with the `map` line 124, I only get the names of the services that I store in the services variable.

Reply:
Oh yes sorry :facepalm:

Reply:
Instead of just having `serviceName: backendName` can we have an object containing `backend: <name>` ?
It will offer more flexibility in the future if we want to add other properties in services description

Reply:
I'm not sure I really understand what you want, you would have something like these examples: `backend: 'Redis Elasticsearch`or an object like this `backend:  { serviceName: backendName, serviceName: backendName, etc ... }`?

Reply:
More something like
```services: {
  storage: {
    backend: 'elasticsearch',
    version: '7.10.2',
  },
  cache: {
    backend: 'redis',
    version: '6.1.0',
  }
}
```

Reply:
I have for the moment displayed the version of each service in hard in the `default.config` until I find a way to retrieve it dynamically